src/panels/lovelace/editor/config-elements/hui-picture-entity-card-editor.ts

```diff
@@ -67,6 +68,7 @@ const SCHEMA = [
     name: "hold_action",
     selector: { "ui-action": {} },
   },
+  { name: "alt_text", selector: { text: {} } },
```

Comment:
Per my other comment, the image can be a function of state, so a single `alt` text is not enough.  Again, we should just build it from the rest of the config.  For example, if the config is an entity with image as a function of state, then we might have ``alt=`${entity.friendly_name}, ${entity.state}` ``.

Reply:
Need to remove the 2 changes from this file as well.